src/extensions/default/SVGCodeHints/SVGAttributes.json

```diff
@@ -11,6 +11,15 @@
     "animate/fill": {
         "attribOptions": ["freeze", "remove"]
     },
+    "animateColor/fill": {
+        "attribOptions": ["freeze", "remove"]
+    },
```

Comment:
I'm not sure what you mean. Locally, I replaced all of the `*/fill` entries with:

```json
    "fill": {
        "attribOptions": ["freeze", "remove"]
    },
```

and it seems to work correctly.